### PR TITLE
Filter mdns candidate if not mdns not enabled

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

File: pkg/rtc/transportmanager.go [hunk 1]
```diff
@@ -424,6 +425,19 @@ func (t *TransportManager) HandleAnswer(answer webrtc.SessionDescription) {
 
 // AddICECandidate adds candidates for remote peer
 func (t *TransportManager) AddICECandidate(candidate webrtc.ICECandidateInit, target livekit.SignalTarget) {
+	if !t.params.Config.UseMDNS {
```
Comment (davidzhao) [inline pkg/rtc/transportmanager.go:428]: nit: should this be in the Transport class instead?

**Review by davidzhao — APPROVED** [review]

File: pkg/rtc/transportmanager.go (hunk 1 above)
Comment (cnderrauber) [reply pkg/rtc/transportmanager.go:428]: Both positions are ok, but filtering it out here can avoid it goes into the transport's event loop

Comment (davidzhao) [reply pkg/rtc/transportmanager.go:428]: sounds good 👍

Comment (boks1971) [reply pkg/rtc/transportmanager.go:428]: I am late to this. If this does not happen too much, I think it is okay to have put it in transport queue and process it there. Would be nice to have all that encapsulated in that one module.

**Pull request merged**